packages/appium/docs/en/guides/migrating-1-to-2.md

```diff
@@ -98,6 +98,13 @@ CLI](../cli/extensions.md) doc)
 
 To update the Appium server itself, you do the same thing as in the past: `npm install -g appium`. Now, installing new versions of the Appium server will leave your drivers intact, so the whole process will be much more quick.
 
+If you would like to update to a specific version, not the latest, please unisntall the driver and install the specific version instead of `update` command.
```

Comment:
typo `unisntall`
and install the specific version using `install` subcommand instead of `update`.